upt_macports/upt_macports.py

```diff
@@ -111,6 +111,14 @@ def _normalized_macports_name(name):
 class MacPortsRubyPackage(MacPortsPackage):
     template = 'ruby.Portfile'
 
+    def _pkgname(self):
+        macports_name = self._normalized_macports_name(self.upt_pkg.name)
+        return macports_name
+
+    @staticmethod
+    def _normalized_macports_name(name):
+        return name
+
```

Comment:
Isn't this a bit over-engineered for now? This could be:

```python
def _pkgname(self):
     return self.upt_pkg.name
```

Also, even if this is simple, it needs unit tests.

Reply:
I think the tests are present now. I agree that this function is not really needed right now, but for the sake of uniformity between code for different frontends, I would be fine with leaving it. If in the end it turns out that we really don't need any conversion of the name we can completely remove it.